pages/en/glossary.mdx

```diff
@@ -0,0 +1,79 @@
+
+- **Subgraph**: A custom API built on blockchain data that can be queried using [GraphQL](https://graphql.org/). Developers can build, deploy and publish subgraphs to The Graph's decentralized network. Then, Indexers can begin indexing subgraphs to make them available to be queried by subgraph consumers.
+
+- **Hosted Service**: A temporary scaffold service for building and querying subgraphs while The Graph's decentralized network matures its cost of service, quality of service, and developer experience.
+
+- **Indexers**: Network participants that run indexing nodes to serve queries to subgraphs.
+
+- **Indexer Revenue Streams**: Indexers are rewarded in GRT with two components: query fee rebates and indexing rewards.
+
+  1. **Query Fee Rebates**: Payments from subgraph consumers for serving queries on the network.
+
+  2. **Indexing Rewards**: The rewards that Indexers receive for indexing subgraphs. Indexing rewards are generated via new issuance of 3% GRT annually.
+
+- **Indexer's Self Stake**: The amount of GRT that Indexers stake to participate in the decentralized network. The minimum is 100,000 GRT, and there is no upper limit.
+
+- **Delegators**: Network participants who own GRT and stake (delegate) their GRT to Indexers. This allows Indexers to increase their stake in subgraphs on the network. In return, Delegators receive a portion of the Indexing Rewards that Indexers receive for processing subgraphs.
+
+- **Delegation Tax**: A 0.5% fee paid by Delegators when they delegate GRT to Indexers. The GRT used to pay the fee is burned.
+
+- **Curators**: Network participants that identify high-quality subgraphs, and “curate” them (i.e., signal GRT on them) in exchange for curation shares. When Indexers claim query fees on a subgraph, 10% is distributed to the Curators of that subgraph. Indexers earn indexing rewards proportional to the signal on a subgraph. We see a correlation between the amount of GRT signalled and the number of Indexers indexing a subgraph.
+
+- **Curation Tax**: A 1% fee paid by Curators when they signal GRT on subgraphs. The GRT used to pay the fee is burned.
+
+- **Subgraph Consumer**: Any application or user that queries a subgraph.
+
+- **Subgraph Developer**: A developer who builds and deploys a subgraph to The Graph's decentralized network.
+
+- **Subgraph Manifest**: A JSON file that describes the subgraph's GraphQL schema, data sources, and other metadata.
+
+- **Rebate Pool**: An economic security measure that holds query fees paid by subgraph consumers until they may be claimed by Indexers as query fee rebates. Residual GRT is burned.
+
+- **Epoch**: A unit of time that in network. One epoch is currently 6646 blocks. 
+
+- **Allocation**: An Indexer can allocate self and delegated stake towards subgraphs that have been published on The Graph's decentralized network. Allocations exist in one of four phases.
```

Comment:
"can allocate self and delegated stake" - sounds a bit weird. "An indexer can allocate their total GRT stake (including delegators stake) " - smth like that

Reply:
THANK YOU! Tried rewording this one a few times and couldn't get it right.